src/codemodder/codemods/upgrade_sslcontext_minimum_version.py

```diff
@@ -0,0 +1,36 @@
+from codemodder.codemods.base_codemod import ReviewGuidance
+from codemodder.codemods.api import SemgrepCodemod
+
+
+class UpgradeSSLContextMinimumVersion(SemgrepCodemod):
+    NAME = "upgrade-sslcontext-minimum-version"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_WITHOUT_REVIEW
+    DESCRIPTION = "Replaces minimum SSL/TLS version for SSLContext"
+
+    @classmethod
```

Comment:
If you want to track an object, use taint mode:
https://semgrep.dev/playground/s/eAoX

See also: limit-readline codemod